frontend/src/app/config/gloves-config.ts

```diff
@@ -6,5 +6,6 @@ import {Config} from './config';
 export const CONFIG: Config = new Config('Blue Gloves 4 All',
     'www.gloveygloversonsgloveemporium.org',
     '/assets/images/banners/gloves-banner.png',
+    '/assets/images/logos/h2ms-logo.png',
```

Comment:
Shouldn't this be a gloves logo?

If this is just a placeholder, please add a TODO